service/http-api-samples/http-api-rv-sample/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright 2020 Intel Corporation
+    SPDX-License-Identifier: Apache 2.0
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>http-api-samples</artifactId>
+    <groupId>org.fido.iot</groupId>
+    <version>0.3-SNAPSHOT</version>
+  </parent>
+
+  <name>HTTP RV API</name>
+  <artifactId>http-api-rv-sample</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <jakson.dataformat.cbor.version>2.11.0</jakson.dataformat.cbor.version>
+    <commons-codec.version>1.14</commons-codec.version>
+    <tomcat.version>9.0.39</tomcat.version>
+    <bcpkix.version>1.65</bcpkix.version>
+    <dbcp2.version>2.7.0</dbcp2.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.fido.iot</groupId>
+      <artifactId>cert-utils</artifactId>
+      <version>0.3-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.fido.iot</groupId>
+      <artifactId>protocol</artifactId>
+      <version>0.3-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.fido.iot</groupId>
+      <artifactId>storage-owner-sample</artifactId>
```

Comment:
Shouldn't this be storage-rv-sample?

Reply:
Considering these are api samples, should the artifact id be 'api-rv-sample'?